post_installation_configuration/node-tasks.adoc

```diff
@@ -159,3 +159,16 @@ include::modules/cluster-node-tuning-operator-default-profiles-set.adoc[leveloff
 include::modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc[leveloffset=+2]
 
 include::modules/nodes-nodes-managing-max-pods-proc.adoc[leveloffset=+1]
+
+[id="nodes-vsphere-scale-static-ip-addresses"]
+== Machine scaling with static IP addresses
+
+After you deployed your cluster to run nodes with static IP addresses, you can scale an instance of a machine or a machine set to use one of these static IP addresses.  
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-installer-infra-requirements_installing-vsphere-installer-provisioned[Static IP addresses for vSphere nodes]
```

Comment:
Need to update this link to point to the level 2 submodule. For the purposes of passing a build for this PR, I went with the xref syntax for the parent module.